MaxText/configs/base.yml

```diff
@@ -214,7 +214,7 @@ logical_axis_rules: [
                        # For pipeline parallelism the pre and post decoder layer tensors' batch dimension is sharded by stages.
                        # Microbatches are sharded by stage, so moving out of and into this sharding should be a local reshape.
                        # The "stage" needs to be listed first since the microbatch dimension is first before the reshape.
```

Comment:
Oh I didn't see this comment, I'll defer to @gobbleturk.

Reply:
This sound be fine, perhaps the order here needs to match the order of mesh axes